README.md

```diff
@@ -1,24 +1,60 @@
-# README
+# MyLibrary
```

Comment:
I think there are a bunch of badges we put on our other repositories that wee might want to insert here as well.

Reply:
```suggestion
# MyLibrary

[![Test Coverage](https://api.codeclimate.com/v1/badges/a8f1c5dab3a53ffba586/test_coverage)](https://codeclimate.com/github/sul-dlss/mylibrary/test_coverage)
```
^-- Here's the codeclimate one

Reply:
Aaaand Travis:
```
[![Build Status](https://travis-ci.org/sul-dlss/mylibrary.svg?branch=master)](https://travis-ci.org/sul-dlss/mylibrary)
```